From 056b41feb6b2f5a1a853c18750c09dc4bde4be1f Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Tue, 20 Jul 2021 12:46:35 +0300
Subject: [PATCH] [SW-45031] gaudi2: add common dir

Added a blank file in arch/gaudi2/common/.
This file should be used for moving functions which
are common between arc, arm.

Change-Id: Id30dc089ff2071bd6711832587572f91faec11dd
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
new file mode 100644
index 0000000..12b111a
--- /dev/null
+++ b/arch/gaudi2/common/common.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ * (C) Copyright 2021 HabanaLabs, Ltd.
+ * Noam Camus <noamikong@gmail.com>
+ *
+ */
+
+/* TODO: Commons functions in arc, arm to be moved here. */
